html/includes/common/availability-map.inc.php

```diff
@@ -42,7 +42,7 @@
             </div>
         </div>';
 
-    if ($config['webui']['availability_map_compact'] == false) {
```

Comment:
I should have looked sooner... this should actually be a comparison with 0 shouldn't it?